test/validation/api/timer/timer.c

```diff
@@ -429,7 +429,7 @@ static void timer_test_timeout_pool_alloc(void)
 
 	/* Try to allocate num items from the pool */
 	for (index = 0; index < num; index++) {
-		odp_event_subtype_t subtype;
+		odp_event_subtype_t subtype = ODP_EVENT_NO_SUBTYPE;
```

Comment:
As before, `ODP_EVENT_PACKET_BASIC` could be used here.

Reply:
Fixed in V2